JWT/src/main/java/io/scalecube/security/SigningKeyResolvers.java

```diff
@@ -57,34 +25,16 @@ private Key retreiveKey(JwsHeader header, Claims claims) {
           return null;
         }
 
-        SignatureAlgorithm algorithm = SignatureAlgorithm.forName(header.getAlgorithm());
         JwtKeyResolver actualResolver = keyResolver.get();
 
-        Map<String, Object> allClaims = Stream.of(claims, (Map<String, Object>) header)
+        Map<String, Object> tokenProperties = Stream.of(claims, (Map<String, Object>) header)
```

Comment:
maybe more simplified

     @Override
      public Key resolveSigningKey(JwsHeader header, Claims claims) {
        return keyResolver.flatMap(actualResolver -> {
          Map<String, Object> tokenProperties = Stream.of(claims, (Map<String, Object>) header)
                  .map(Map::entrySet)
                  .flatMap(Collection::stream)
                  .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
          return actualResolver.resolve(tokenProperties);
        }).orElse(null);
      }